make/makefile

```diff
@@ -40,14 +40,18 @@ I= -I$(INCL) -I$S/include/
 TO_OS?= TO_LINUX
 OS_ID?= 0.4.4
 BIN_SUFFIX=
-RAPI_FLAGS=  -O2 -fvisibility=hidden -m32
-HOST_FLAGS=	-DREB_EXE  -O2 -fvisibility=hidden -m32 -D_FILE_OFFSET_BITS=64
 RLIB_FLAGS=
 
+#RAPI_FLAGS= -g -O0 -UNDEBUG -std=c99 -pedantic -Werror -Wextra -Wno-long-long -Wno-unused-parameter -fvisibility=hidden
+#HOST_FLAGS= -g -O0 -UNDEBUG -std=c99 -pedantic -Werror -Wextra -Wno-long-long -Wno-unused-parameter -fvisibility=hidden -DREB_EXE -D_FILE_OFFSET_BITS=64
+
+RAPI_FLAGS= -x c++ -g -O0 -std=c++14 -UNDEBUG -pedantic -Wextra -Wno-long-long -Wno-unused-parameter -fvisibility=hidden
```

Comment:
Accidentally committed one of the makefiles here.  Will squash it out eventually, but note that controlling a debug build is now done in the typical C way with -DNDEBUG.  If you don't define NDEBUG the asserts will be compiled in.